Templates : déplacement de l'appel au sous programme Image_MF + petits correctifs

dominique95880 opened the pull request:
- Depuis la disparition du second paramètre du sous programme template
image_MF, il est souhaitable que tous les appels à ce sous-programme
soient faits le plus haut possible dans les divers composants des
templates.
- Remplacement de l'image picto-molette.png par une class
defont-awesome.min.css
- Rectification VR en VD pour Vendémiaire dans les formulaires de MAJ
updind.txt et updfam.txt

Voir #553 et #517

dominique95880 committed:
Templates : déplacement de l'appel au sous programme Image_MF + petits correctifs

- Depuis la disparition du second paramètre du sous programme template
image_MF, il est souhaitable que tous les appels à ce sous-programme
soient faits le plus haut possible dans les divers composants des
templates.
- Remplacement de l'image picto-molette.png par une class
defont-awesome.min.css
- Rectification VR en VD pour Vendémiaire dans les formulaires de MAJ
updind.txt et updfam.txt

a2line reviewed (commented):
Sur la chronologie il y a maintenant des symboles en double pour les naissances/décès des parents/enfants/épouse à chasser.

![image](https://cloud.githubusercontent.com/assets/10353895/25579947/bca10f0e-2e7c-11e7-910b-7d2206ef77fe.png)

Sur les arbres, par construction on connait le sexe et on a pas vraiment besoin du symbole. Si on veut le garder sur anctree avant les prénoms (et non se promenant parfois au-dessus comme le rendu actu), il faudrait réussir à rendre l'ensemble icone+prénom non sécable.

![image](https://cloud.githubusercontent.com/assets/10353895/25579942/a8fe76ee-2e7c-11e7-9332-17b4b56afa58.png)

dominique95880 commented:
- Pour la chronologie, on peut simplifier l'écriture en virant les appels à image_MF. (Je peux m'en charger)
- Pour les arbres, comme il peut y avoir inversion des sexes ou mariage homosexuel, je maintiens mon point de vue.
OK pour l'aspect non sécable avant le prénom, ça doit pouvoir se résoudre avec le caractère html `&nbsp;`
- Pour le côté "on a besoin que l'arbre miniature ne soit pas trop large", c'est deux caractères !
- Quant à "Pas besoin du symbole ici en h1", c'est le seul moyen de savoir si l'individu est un homme ou une femme (ou un neutre).
Les couleurs des icônes de menubar ne sont qu'un plus (je ne les avais même pas vues).
Si le symbole est trop gros, on peut le réduire en utilisant une classe supplémentaire de font-awesome.min.css prévue à cet effet ou un size=50%.

dominique95880 committed:
Templates : Déplacement de l'appel au sous programme Image_MF (suite)

Suppression des appels à image_MF dans perso_module\chronologie.txt
faisant double emploi avec les appels faits dans perso_utils.txt
Voir #555

dominique95880 committed:
main trop lourde dans modif chronologie.txt

GuillaumeBrochu commented:
Est-ce que cette PR est prête à être intégrée?

dominique95880 commented:
Pour moi OUI

hgouraud commented:
Si je prend le seul exemple de short_display_person, il y a aujourd'hui beaucoup d'endroits où il est appelé sans être précédé par image_MF. Souhaite t'on avoir systématiquement ce symbole avant le nom des personnes? Il y a en particulier des cas qui ne fonctionneront pas 
 exemple : [*on %s's side:::] %apply;short_display_person("father")
on ne veut pas de symbole avant le nom!! de plus, cela détruirait la possibilité de l'élision!!

hgouraud commented:
Par contre, le correctif VR -> VD est à faire. Le sortir de cette PR serait bien.
